ui/src/components/view/UploadResourceIcon.vue

```diff
@@ -195,7 +195,7 @@ export default {
       this.options.img = ''
     },
     beforeUpload (file) {
-      if (!/\.(svg|jpg|jpeg|png|bmp|SVG|JPG|PNG)$/.test(file.name)) {
+      if (!/\.(svg|jpg|jpeg|png|bmp|SVG|JPG|JPEG|PNG)$/.test(file.name)) {
```

Comment:
@DaanHoogland 

```suggestion
      if (!/\.(bmp|jpeg|jpg|png|svg)$/i.test(file.name)) {
```
Ordered the file extensions and used a case insensitive regex

Reply:
@kiranchavala can you agree/apply this suggestion?

Reply:
@DaanHoogland yes making it case insensitive is good thought

Reply:
Tested the change internally and it working fine with case-insensitive regex

Reply:
than pleae apply the change @kiranchavala

Reply:
@DaanHoogland  applied the changes